common/build.gradle

```diff
@@ -1,3 +1,12 @@
+repositories {
```

Comment:
Can you remove this section? The VS2 Maven will automatically shade the repository when it builds on github.

Reply:
Sadly, when I do, IntelliJ yells at me when I refresh Gradle